templates/appveyor.yml

```diff
@@ -27,4 +30,5 @@ before_test:
 test_script:
   - node --version
   - npm --version
-  - cmd: npm run ci:%job_part%
+  - cmd: FOR /F %%I in ('compver --name webpack --gte %webpack_version% --lt latest') do SET COMPARED_VERSION_RESULT=%%I
+  - cmd: IF %COMPARED_VERSION_RESULT% NEQ -1 (npm run ci:%job_part%) ELSE (ECHO "Next is older than Latest - Skipping Canary Suite")
```

Comment:
We skip canary test if canary older than latest as for CircleCI

Reply:
```diff
- "Next is older than Latest - Skipping Canary Suite"
+ `webpack@next is older than webpack@latest - Skipping Canary Suite"
```

How do we achieve the same for CircleCI ? We also need to clearify if the `webpack@next` pattern is still be used by `webpack/webpack`

Reply:
@michael-ciniawsky It is same message as in `Circle CI`. Really? There will be no one ever read.  

> We also need to clearify if the webpack@next pattern is still be used by webpack/webpack itself
- Yes.
- And we should in any case correct this for the current version, even if in the end it will be deleted (major release).

Reply:
You can keep it as-is as it is the same message as in the CircleCI config... If nobody ever reads it why is it there in the first place ? 🙃 (That was my first intend to just propose to remove it as long as the check reliably works)

Reply:
@michael-ciniawsky in near future `webpack` will starting release `webpack@5` as `next` and we can catch problems first, merge this as is